### PR TITLE
Try to use different file superclass

### PR DESCRIPTION
**Pull request by martindurant**
because HTTPfile requires aiohttp

### CODE REVIEW

File: dropboxdrivefs/core.py [hunk 1]
```diff
@@ -148,14 +148,15 @@ def __init__(
...
+
         Parameters
         ----------
         fs: instance of DropboxDriveFileSystem
         path : str
             file path to inspect in dropbox
         mode: str
-            Normal file modes.'rb', 'wb' or 'ab'
+            most be "wb"
         block_size: int or None
```
Comment (MarineChap) [inline dropboxdrivefs/core.py:173]: What happened with the read mode ?

Comment (martindurant) [reply dropboxdrivefs/core.py:173]: In `_open`, read mode always picks a different class.

**Comment by martindurant** [comment]
@rabernat ,  @annieleal with the latest push, you are able to open the zip file

File: dropboxdrivefs/core.py (hunk 1 above)
Comment (MarineChap) [reply dropboxdrivefs/core.py:173]: ah yes of course.

File: dropboxdrivefs/core.py [hunk 2]
```diff
@@ -22,10 +22,23 @@ class DropboxDriveFileSystem(AbstractFileSystem):
 
     def __init__(self, token, *args, **storage_options):
         super().__init__(token=token, *args, **storage_options)
-        logging.basicConfig(format="%(levelname)s:%(message)s", level=logging.INFO)
```
Comment (MarineChap) [inline dropboxdrivefs/core.py:25]: why did you remove this ?

**Comment by martindurant** [comment]
@MarineChap , to transfer ownership to `fsspec`, you can do this in the settings portion of the repo, or you can instead give me admin rights, and I can move it. You would remain admin of this repo regardless.

**Comment by MarineChap** [comment]
I don't have the right to create public repo in fsspec orga when I try to transfer the ownership. 
I added you as a collaborator but as I don't have an organization account, I don't think there is more I can do. Do you have enough access to do the transfer ?

**Comment by martindurant** [comment]
Hm, I still can't do it - I suppose I can't be "owner" of your private (as opposed to org) repo. I'll add you to the fsspec org, then you can do it.

**Comment by MarineChap** [comment]
Ok it is officially done. :) 

Also, should we merge this or you are waiting for other feedbacks ?

**Comment by martindurant** [comment]
I am waiting on @rabernat and others to ensure this actually works.

**Comment by rabernat** [comment]
Here is my completely copy-pasteable reproducible example

```python
import dropbox
import dropboxdrivefs as dbx
import fsspec
from fsspec.implementations.zip import ZipFileSystem

token = 'u19t2wI3eH4AAAAAAAAAAdplJXSWJIRd1Rp8sAl1MIcxTxz4fkJ8xP_y8dqS7sdv'
path = '/RN-20211117121141_CF2D9BA9D51D3B7CE0538486ABC0F5FD/idp2021/GEOTRACES_IDP2021_v1_seawater_netcdf.zip'

dfs = dbx.DropboxDriveFileSystem(token=token)
dbox_file = dfs.open(path)
zfs = ZipFileSystem(dbox_file)
```

When I try with this branch, I get the following error (also reported in https://github.com/fsspec/filesystem_spec/issues/844#issuecomment-987014719)

```
---------------------------------------------------------------------------
ValidationError                           Traceback (most recent call last)
/tmp/ipykernel_426/254540335.py in <module>
      8 
      9 dfs = dbx.DropboxDriveFileSystem(token=token)
---> 10 dbox_file = dfs.open(path)
     11 zfs = ZipFileSystem(dbox_file)

~/.local/lib/python3.8/site-packages/fsspec/spec.py in open(self, path, mode, block_size, cache_options, **kwargs)
    960         else:
    961             ac = kwargs.pop("autocommit", not self._intrans)
--> 962             f = self._open(
    963                 path,
    964                 mode=mode,

/srv/conda/envs/notebook/lib/python3.8/site-packages/dropboxdrivefs/core.py in _open(self, path, mode, block_size, autocommit, cache_options, **kwargs)
    110         if mode == "rb":
    111             url = self.dbx.files_get_temporary_link(path).link
--> 112             return webhdfs.WebHDFile(
    113                 self,
    114                 url,

~/.local/lib/python3.8/site-packages/fsspec/implementations/webhdfs.py in __init__(self, fs, path, **kwargs)
    348 
    349     def __init__(self, fs, path, **kwargs):
--> 350         super().__init__(fs, path, **kwargs)
    351         kwargs = kwargs.copy()
    352         if kwargs.get("permissions", None) is None:

~/.local/lib/python3.8/site-packages/fsspec/spec.py in __init__(self, fs, path, mode, block_size, autocommit, cache_type, cache_options, **kwargs)
   1288         if mode == "rb":
   1289             if not hasattr(self, "details"):
-> 1290                 self.details = fs.info(path)
   1291             self.size = self.details["size"]
   1292             self.cache = caches[cache_type](

/srv/conda/envs/notebook/lib/python3.8/site-packages/dropboxdrivefs/core.py in info(self, url, **kwargs)
     94         """Get info of URL
     95         """
---> 96         metadata = self.dbx.files_get_metadata(url)
     97         return self._refactor_metadata(metadata)
     98 

/srv/conda/envs/notebook/lib/python3.8/site-packages/dropbox/base.py in files_get_metadata(self, path, include_media_info, include_deleted, include_has_explicit_shared_members, include_property_groups)
   1501             :class:`dropbox.files.GetMetadataError`
   1502         """
-> 1503         arg = files.GetMetadataArg(path,
   1504                                    include_media_info,
   1505                                    include_deleted,

/srv/conda/envs/notebook/lib/python3.8/site-packages/dropbox/files.py in __init__(self, path, include_media_info, include_deleted, include_has_explicit_shared_members, include_property_groups)
    168         self._include_property_groups_value = bb.NOT_SET
    169         if path is not None:
--> 170             self.path = path
    171         if include_media_info is not None:
    172             self.include_media_info = include_media_info

/srv/conda/envs/notebook/lib/python3.8/site-packages/stone/backends/python_rsrc/stone_base.py in __set__(self, instance, value)
     79             self.validator.validate_type_only(value)
     80         else:
---> 81             value = self.validator.validate(value)
     82         setattr(instance, self.name, value)
     83 

/srv/conda/envs/notebook/lib/python3.8/site-packages/stone/backends/python_rsrc/stone_validators.py in validate(self, val)
    343 
    344         if self.pattern and not self.pattern_re.match(val):
--> 345             raise ValidationError("'%s' did not match pattern '%s'"
    346                                   % (val, self.pattern))
    347         return val

ValidationError: 'https://ucd838635c7e3e5c25fa00682efd.dl.dropboxusercontent.com/cd/0/get/BbaUQjUdBVA31tj28XeMFHm7gA-jnRliMUJdU1eb3oQHZeOytS8fBtn0qIvetBfIwLxgrnzJVij6Vjp1WyCFXhh5eNe54bdpo8nUhMxPfk2_75STq4Ub6bdj5AXENLcz-YSqbjjkRE3Q-i0TvDzmfLDE/file' did not match pattern '(/(.|[\r\n])*|id:.*)|(rev:[0-9a-f]{9,})|(ns:[0-9]+(/.*)?)'
```

Martin, does it work for you?

File: dropboxdrivefs/core.py (hunk 2 above)
Comment (rabernat) [reply dropboxdrivefs/core.py:25]: Logging should be configured by the user, not set by the package itself.

**Comment by martindurant** [comment]
Yes, the snippet works for me at a581055 and fsspec master

**Comment by martindurant** [comment]
( ping @rabernat - yes, it worked)
